reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md

```diff
@@ -1,47 +1,35 @@
 ---
 title: What's New in PowerShell 7.1
 description: New features and changes released in PowerShell 7.1
-ms.date: 11/11/2020
+ms.date: 12/15/2020
 ---
 
 # What's New in PowerShell 7.1
 
-PowerShell 7.1 is an open-source, cross-platform (Windows, macOS, and Linux) edition of PowerShell,
-built to manage heterogeneous environments and hybrid cloud.
-
 Building on the foundation established in PowerShell 7.0, our efforts focused on community issues
 and include a number of improvements and fixes. We are committed to ensuring that PowerShell remains
 a stable and performant platform.
 
-PowerShell 7.1 also include PSReadLine 2.1.0. This version includes Predictive IntelliSense. For
-more information about the Predictive IntelliSense feature, see the
-[announcement](https://devblogs.microsoft.com/powershell/announcing-psreadline-2-1-with-predictive-intellisense/)
-in the PowerShell blog.
-
-## Where can I install PowerShell?
-
-<!-- TODO: Update list of OS below - make sure this is consistent across all docs -->
+PowerShell 7.1 includes the following features, updates, and breaking changes.
 
-PowerShell 7.1 currently supports the following operating systems on x64, including:
+- PSReadLine 2.1.0, which includes Predictive IntelliSense
+- PowerShell 7.1 has been published to the Microsoft Store
+- Installer packages updated for new OS versions with support for ARM64
+- 4 new experimental features and 2 experimental features promoted to mainstream
+- Several breaking changes to improve usability
 
-- Windows 8.1/10 (including ARM64)
-- Windows Server 2012 R2, 2016, 2019, and Semi-Annual Channel (SAC)
-- Ubuntu 16.04/18.04/20.04 (including ARM64)
-- Ubuntu 19.10 (via Snap package)
-- Debian 9/10
-- CentOS and RHEL 7/8
-- Fedora 32
-- Alpine 3.11+ (including ARM64)
-- macOS 10.13+
+For a complete list of changes, see the
+[CHANGELOG](https://github.com/PowerShell/PowerShell/blob/master/CHANGELOG/7.1.md) in the GitHub
+repository.
 
-We also have community support for:
+## PSReadLine 2.1.0
 
-- Arch Linux
-- Raspbian Linux
-- Kali Linux
+PSReadLine 2.1.0, which includes PowerShell 7.1 also include PSReadLine 2.1.0. This version includes
```

Comment:
Might be just me, but this sentence is a little confusing?

Reply:
I thought I fixed this already.